fedora-python-bootstrapper.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: all_vms
+  become: true
+  become_user: root
+  gather_facts: False
+  tasks:
+    # https://trello.com/c/XaiXEocS/239-bz-to-track-adding-python-to-the-fedora-cloud-images
+    - name: install python2 and dnf stuff
+      raw: (dnf -y install python-dnf python2-dnf libselinux-python)
```

Comment:
Is there a reason you're not using a `with_items` and `package` module here?

Reply:
Indeed, raw is required for this particular play because the Fedora cloud images don't have Python, so... without Python, raw is the only way to get Python in there.

Reply:
Really? Hrmmmm that's weird... I don't think any of my CentOS images have Python installed by default either?